src/components/FilterButton/FilterButton.jsx

```diff
@@ -2,20 +2,24 @@ import { useState } from "react";
 
 export const FilterButton = ({filterOrigin, origin}) => {
   const [buttonColor, setButtonColor] = useState('white');
+  const [buttonSelected, setButtonSelected] = useState(false);
+
   const markButtonSelected = () => {
-    buttonColor === "white" ? setButtonColor("lightBlue") : setButtonColor("white");
+    buttonSelected ? setButtonColor("white") : setButtonColor("lightBlue");
   }
+
   return (
     <button key={origin} className="filterButton"
-    key={origin}
-    onClick={() => {
-    filterOrigin({origin}); 
-    markButtonSelected();
-  }} 
-  style={{
-    backgroundColor: buttonColor,
-    borderRadius: '8px',
-    marginBottom: '20px'
-  }}>{origin}</button>
+      onClick={() => {
+        const isButtonSelected = setButtonSelected(!buttonSelected);
+        filterOrigin({origin, isButtonSelected}) 
+      markButtonSelected();
+      }} 
+      style={{
```

Comment:
avoid the use of inline styles at all costs

Reply:
Removed